core/workspace_svg.ts

```diff
@@ -1382,10 +1391,6 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
             for (let i = 0, connection; (connection = connections[i]); i++) {
               const neighbour = connection.closest(
                 config.snapRadius,
-                // TODO: This code doesn't work because it's passing an absolute
-                //     coordinate instead of a relative coordinate. Need to
-                //     figure out if I'm deprecating this function or if I
-                //     need to fix this.
```

Comment:
Any particular reason to delete these comments (here and blelow)?  Deciding to deprecate has (presumably) not caused the code to become correct…

Reply:
True! The TODO was to decide to fix it or not, and I decided not to so I deleted the TODO :P But I can leave a comment that the code is incorrect.

Reply:
Good point re "TODO", but yes let's keep documentation of bugs until the buggy code is actually deleted.